scripts/pyro_camera_server.py

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env python
 """
-Script to run a Pyro camera server. This should be run on the control computer for a distributed
-camera. The configuration for the camera is read from the pyro_camera.yaml/pyro_camera_local.yaml
-config file. The camera servers should be started after the name server, but before POCS.
+Script to run a Pyro camera server. This should be run on the camera control computer for a
+distributed camera. The configuration for the camera is read from the pyro_camera.yaml/
+pyro_camera_local.yaml config file. The camera servers should be started after the name server,
```

Comment:
Well, for now it involves a bunch of ssh sessions. My plan though is to remove the ordering requirement.

Need to get the `Observatory` to periodically check the name server and update its list of distributed cameras, rather than just doing this once as a result of the call of `create_cameras`. Similarly need to get the camera servers to check the name server and make sure they're registered, and also deregister on shutdown. With those modifications it would be possible to (re)start camera servers after starting POCS and, with a bit of care, allow the name server to be restarted while POCS is running too.